sass/true/_results.scss

```diff
@@ -31,6 +34,11 @@
     $message: _true-report-message('global', 'single');
     @include _true-message($message, 'debug');
   }
+
+  @if $stats {
```

Comment:
Since `true-terminal-output` is a global setting phrased generically, it should probably have an impact on stats output as well? Or we should re-think the settings?